client/components/LearnerViewDirective.js

```diff
@@ -71,6 +50,27 @@ tie.directive('learnerView', [function() {
               </div>
             </div>
           </div>
+          <div class="tie-question-ui">
+            <div class="tie-question-window">
+              <h3>Exercise {{exerciseNumber}}: {{title}}</h3>
```

Comment:
In that case, I disagree. I think that "Exercise 1" is a property of the sequence of exercises, and not a property of the individual exercise. We do not have a data-driven way to represent a sequence of lessons yet, so we'll need to handle that in code until we can factor it out (similar to my other comment about the focus point of the sequence).